honeybee_grasshopper_core/src/HB Run Recipe.py

```diff
@@ -107,24 +107,27 @@ def sort_results_by_grid(results, grid_names):
     # write the inputs JSON for the recipe
     inputs_json = _recipe.write_inputs_json(_folder_)
 
-    # execute the queenbee luigi CLI to obtain the results via CPython
-    queenbee_exe = os.path.join(hb_folders.python_scripts_path, 'queenbee.exe') \
-        if os.name == 'nt' else os.path.join(hb_folders.python_scripts_path, 'queenbee')
-    cmds = [queenbee_exe, 'luigi', 'translate', _recipe.path, _folder_,
-            '-i', inputs_json, '--workers', _cpu_count_]
-    if rad_folders.radlib_path:  # set the RAYPATH environment variable
-        cmds.extend(['--env', 'RAYPATH={}'.format(rad_folders.radlib_path)])
-    if rad_folders.radbin_path:  # set the PATH environment variable
-        cmds.extend(['--env', 'PATH={}'.format(rad_folders.radbin_path)])
-    cmds.append('--run')
+    genv = os.environ.copy()
+    genv['PATH'] = os.pathsep.join((rad_folders.radbin_path, genv['PATH']))
+    genv['RAYPATH'] = os.pathsep.join((rad_folders.radlib_path, genv['RAYPATH']))
```

Comment:
Could you put a comment here that explains what's going on? The `genv` doesn't look like it's passed to anything else so it's a little tough to know how it affects the rest of the code.

Reply:
It's a mistake and thank you for catching it. We should just pass these two as --env and they will be added to the env inside the command: https://github.com/ladybug-tools/honeybee-radiance-recipe/pull/43/files#diff-19a8e782c877d36f71a03325888aefd0deb331d80ff00a479aeaa6c736349c81R74-R100